### PR TITLE
Add support for feature tests running with uppercase F

### PR DESCRIPTION
**Pull request by RDIL**
# PR Summary
Not sure if this is done already, but noticed lots of people use an uppercase `F` so thought this may be worth pointing out

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [x] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [x] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by vexx32** [comment]
@RDIL `-match` in PS is case-insensitive by default, so if there is a casing limitation, it's not imposed here. 🙂

**Comment by RDIL** [comment]
Ah ok

File: tools/ci.psm1 [hunk 1]
```diff
@@ -97,12 +97,12 @@ Function Test-DailyBuild
...
     }
 
-    # if [feature] is in the commit message,
+    # If [feature] or [Feature] is in the commit message,
     # Run Daily tests
     $commitMessage = Get-CommitMessage
     Write-log -message "commitMessage: $commitMessage"
 
-    if($commitMessage -match '\[feature\]' -or $env:FORCE_FEATURE -eq 'True')
+    if($commitMessage -match '\[feature\]' -or $commitMessage -match '\[Feature\] -or $env:FORCE_FEATURE -eq 'True')
```
Comment (iSazonov) [inline tools/ci.psm1]: `-match` is case-insensitive and we don't need the change.

Comment (RDIL) [reply tools/ci.psm1]: thats why I closed it

**Pull request closed without merging**